.gitignore

```diff
@@ -1,4 +1,4 @@
-# Compiled Object files
+ Compiled Object files
```

Comment:
Will removing the leading '#' cause any problems?

Reply:
Not sure, but removing it was an accident so I've undone it.